mysql/datadog_checks/mysql/data/conf.yaml.example

```
@@ -86,37 +86,32 @@ instances:
-    ## * 'false' - `custom_queries` override `global_custom_queries`.
-    ## * 'extend' - `global_custom_queries` are used in addition to any `custom_queries`.
+    ## 1. true - `global_custom_queries` override `custom_queries`.
+    ## 2. false - `custom_queries` override `global_custom_queries`.
+    ## 2. extend - `global_custom_queries` are used in addition to any `custom_queries`.
     #
     # use_global_custom_queries: 'true'
 
     ## @param custom_queries - list of mappings - optional
-    ## A list of custom queries used to retrieve custom metrics.
-    ##
-    ## The following fields are expected:
-    ##
-    ## query:
-    ##   The SQL query to execute. It can be a simple statement, or a multi-line script.
-    ## columns:
-    ##   A list representing each column, ordered sequentially from left to right.
-    ##   The number of columns must be equal to the number of columns returned in the `query`.
-    ##
-    ##   Each column is an object with the following fields:
+    ## Each query must have 2 fields:
     ##
-    ##   name:
-    ##     The suffix to append to `mysql.` to form the full metric name.
-    ##     If `type` is `tag`, this column is considered a tag and applied to every metric
-    ##     collected by this query.
-    ##   type:
-    ##     The submission method (gauge, monotonic_count, etc.).
-    ##     Use the 'tag' type to tag each metric in the row with the name and value
-    ##     of the item in this column.
-    ##     Use the 'count' type to perform aggregation for queries that return multiple
-    ##     rows with the same or no tags.
-    ## tags: (optional)
-    ##   A list of tags to apply to each metric.
+    ## 1. query - The SQL to execute. It can be a simple statement or a multi-line script.
+    ##            Use the pipe `|` if you require a multi-line script.
+    ## 2. columns - The list representing each column, ordered sequentially from left to right.
+    ##              The number of columns must equal the number of columns returned in the query.
+    ##              There are 2 required pieces of data:
+    ##                a. name - The suffix to append to `clickhouse.` to form
+    ##                          the full metric name. If `type` is `tag`, this column is
+    ##                          considered a tag and applied to every
+    ##                          metric collected by this particular query.
+    ##                b. type - The submission method (gauge, monotonic_count, etc.).
+    ##                          This can also be set to `tag` to tag each metric in the row
+    ##                          with the name and value of the item in this column. You can
+    ##                          use the `count` type to perform aggregation for queries that
+    ##                          return multiple rows with the same or no tags.
+    ##              Columns without a name are ignored, use this for any column you wish to skip:
```

```suggestion
    ##              Columns without a name are ignored. To skip a column, enter:
```
At least, I think that's what you mean?